### DISCUSSION TITLE
Publish `amqp_client` 3.13.x GA versions to hex

### DISCUSSION THREAD

**Discussion by bryannaegele** [rabbitmq/rabbitmq-server#11204] [Other]
### Is your feature request related to a problem? Please describe.

The hex package hasn't been marked stable for 3.13.x yet. It also doesn't appear to have been published with dependency on a stable `rabbit_common` 3.13.x version.

### Describe the solution you'd like

Publish `amqp_client` 3.13.x non-rc versions

**Comment by michaelklishin** [comment]
Our team is currently going through a lot of drastic infrastructure changes (since VMware, Inc. has been acquired by Broadcom, Inc.) and I cannot promise when the pipeline (well, a part of the server pipeline) that publishes the Erlang AMQP 0-9-1 client will be ready again to publish to Hex.pm. It can take a couple of months.

Rebar 3 surely allows you to consume a dependency from Git using a tag or commit. So I suggest that you do that for the foreseeable future.

Or someone can work on setting up an Actions workflow for publishing `amqp_client` releases but even then, sadly reviewing and testing it won't be a priority. RabbitMQ's own use of the client is growing slimmer over time (we use internal APIs instead where possible) and most paying users do not use Erlang.

Plus I don't recall any meaningful AMQP 0-9-1 Erlang client changes recently. In the AMQP 1.0 Erlang client there were some, and its importance keeps growing.

**Reply by bryannaegele** [reply]
Good to know. If there aren't any breaking changes that would prevent using it with 3.13 then it's not an issue to continue with the 3.12.x versions.

I was just confused that 3.12.x versions have continued to be published along with those releases but 3.13 client just hadn't been moved from RC when the server got release, so I figured it was just an oversight. `rabbit_common` 3.13.x has also been getting published.

**Reply by michaelklishin** [reply]
You can use the 3.12.x Erlang AMQP 0-9-1 client with RabbitMQ 3.13.x nodes. Certainly if you use the network client that uses TCP but even the direct client that uses Erlang distribution is likely perfectly fine with the 3.13.x versions of `rabbit_direct`, a module it relies on.
